CMakeLists.txt

```diff
@@ -1,2 +1,7 @@
 cmake_minimum_required (VERSION 3.1 FATAL_ERROR)
+
+if (NOT DEFINED MEDIA_RUN_TEST_SUITE)
```

Comment:
Because GmmLib is generic library between Intel Media driver and Intel Compute-runtime, I will keep the naming generic as RUN_TEST_SUITE.

Reply:
ok, ftr the name was a request from Media driver devs.